atd-vze/src/views/Reports/CrashesInconsistent.js

```diff
@@ -0,0 +1,45 @@
+import React from "react";
+import { withApollo } from "react-apollo";
+
+import GridTable from "../../Components/GridTable";
+import gqlAbstract from "../../queries/gqlAbstract";
+
+import {
+  crashGridTableColumns,
+} from "./crashesInconsistentTableParameters";
+
+// Our initial query configuration
+let queryConf = {
+  table: "view_crashes_inconsistent_numbers",
```

Comment:
Should we add the SQL code for this view table to the codebase?

Reply:
Good call, forgot about it!